tests_integ/helpers/deployer/utils/table_print.py

```diff
@@ -0,0 +1,137 @@
+"""
```

Comment:
Do we need this? (Or the colors class?)

Reply:
It can be useful to debug a test error and it isn't displayed if a test succeeds